src/main/java/com/faforever/api/security/OAuthScope.java

```diff
@@ -0,0 +1,47 @@
+package com.faforever.api.security;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public enum OAuthScope {
+
+  PUBLIC_PROFILE("public_profile", "Read your public player data"),
+  READ_ACHIEVEMENTS("read_achievements", "Read your achievements"),
+  WRITE_ACHIEVEMENTS("write_achievements", "Write your achievements"),
+  READ_EVENTS("read_events", "Read events"),
+  WRITE_EVENTS("write_events", "Write events"),
+  UPLOAD_MAP("upload_map", "Upload maps"),
+  UPLOAD_MOD("upload_mod", "Upload mods"),
+  WRITE_ACCOUNT_DATA("write_account_data", "Edit account data"),
+  EDIT_CLAN_DATA("edit_clan_data", "Edit clan data");
```

Comment:
That's the problem; you can't use properties of enums in annotations. We have to use String constants.